### ISSUE TITLE
Dataflow and immutable collections allegedly "not supported" in .netstandard

### ISSUE THREAD

**Issue by AArnott** [Microsoft/dotnet-apiport#400]
All my uses of System.Threading.Tasks.Dataflow and System.Collections.Immutable are flagged as "Not supported" but of course they are, because these are .netstandard libraries.

**Comment by Petermarcu** [comment]
Yeah, this is an ongoing request I believe. We've hit it too. The problem is the tool currently only considers what is "in the platform" and not all the things that can run just fine "on the platform".

**Comment by cartermp** [comment]
@Petermarcu have there been any changes on this front?  This affects F# all-up, as it will report that all F# code is incompatible because FSharp.Core is not "in the platform", but a separate `.dll` (or package for Core/Standard).

**Comment by Petermarcu** [comment]
@twsouthwick , @conniey , can you give an update on where we are with including more of the ecosystem that supports .net standard? That's critical as more libraries are converted. F# also appears to need it.

**Comment by conniey** [comment]
We fetch our API information from the catalog generated from @terrajobst..  @chlowell and I had looked into using NuGet package data to add more data to the ecosystem, but the Kusto data was missing DocIds, which is what we use.

**Comment by twsouthwick** [comment]
Is there a new identifier that is being used? I noticed that with the new doc system, docids have been replaced with something else.
